### ISSUE TITLE
Is it support android? 

### ISSUE THREAD

**Issue by Parkjonghyun93** [ontio-community/ontology-andriod-sdk#78]
Hi :)
I fixed the gradle by copying the project (https clone).
problems..
 1. There is no file to build. 

```
import com.github.ontio.sdk.exception.SDKRuntimeException;
                                     ^
  symbol:   class SDKRuntimeException

in WasmScriptBuilder class 

```
```

super(TransactionType.InvokeWasm);
                             ^
  symbol:   variable InvokeWasm

in InvokeWasmCode class 

```

 2. Some functions are not overridden.
```

error: DeployWasmCode is not abstract and does not override abstract method getAddressU160ForVerifying() in Signable
public class DeployWasmCode extends Transaction 
```

What am I missing? 

(It is deprecated library??)

**Comment by lucas7788** [comment]
sorry, it is our bug, we have fix it with #80  @Parkjonghyun93

**Comment by Parkjonghyun93** [comment]
Thanks for your feedback and i checked it
